core/test/equations.jl

```diff
@@ -96,12 +90,12 @@ end
     t = Ribasim.timesteps(model)
     storage = Ribasim.get_storages_and_levels(model).storage[1, :]
     basin_area = p.basin.area[1][2]
-    storage_min = 50.0
+    storage_min = 50.005
```

Comment:
It turns out for the test to pass it is not necessary. This stems from when the area of the profile had to start at 0, which means that towards the bottom the area cannot be constant. So I made these tests so that after this minimum storage the area is constant, which makes it easier to obtain an analytical solution.